lib/Time/gmtime.pm

```diff
@@ -1,27 +1,24 @@
 package Time::gmtime;
-use strict;
-use 5.006_001;
+use v5.38;
+no feature 'signatures'; # we use prototypes
 
-use Time::tm;
+use parent 'Time::tm';
 
-our (@ISA, @EXPORT, @EXPORT_OK, %EXPORT_TAGS, $VERSION);
 our (   $tm_sec, $tm_min, $tm_hour, $tm_mday,
         $tm_mon, $tm_year, $tm_wday, $tm_yday, 
 		$tm_isdst,
 );
- 
-BEGIN { 
-    use Exporter   ();
-    @ISA         = qw(Exporter Time::tm);
-    @EXPORT      = qw(gmtime gmctime);
-    @EXPORT_OK   = qw(  
+
+use Exporter 'import';
+our @ISA         = qw(Exporter Time::tm);
```

Comment:
Surely you don't want this anymore? `parent` will do it's thing and we're importing Exporter's import now rather than subclassing it.